test/soca/CMakeLists.txt

```diff
@@ -0,0 +1,176 @@
+# Copy the bkg files
+# ------------------
+set(TESTDATA ${PROJECT_BINARY_DIR}/test/testdata )
+
+
+# Symlink test input yaml files
+# -----------------------------
+# create testinput dir
+set (TESTINPUT_DIR ${PROJECT_BINARY_DIR}/test/soca/testinput)
+
+file(MAKE_DIRECTORY ${TESTINPUT_DIR})
+
+# symlink
+foreach(FILENAME ${test_input})
+	get_filename_component(filename ${FILENAME} NAME)
+	execute_process( COMMAND ${CMAKE_COMMAND} -E create_symlink
+		${FILENAME}
+		${TESTINPUT_DIR}
+	)
+endforeach(FILENAME)
+
+# install
+install(FILES ${test_input}
+        DESTINATION "test/testinput/")
+
+# Test exgdas scripts from the global-worflow
+if (WORKFLOW_TESTS)
+  add_subdirectory(gw)
+endif()
+
+# test for ush/socaincr2mom6.py
+add_test(NAME test_gdasapp_soca_nsst_increment_to_mom6
+         COMMAND ${PROJECT_SOURCE_DIR}/test/soca/socaincr2mom6.sh ${PROJECT_SOURCE_DIR}
+         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/test/soca)
+set_tests_properties(
+        test_gdasapp_soca_nsst_increment_to_mom6
+PROPERTIES
+ENVIRONMENT "data=${TESTDATA};PYTHONPATH=${PROJECT_BINARY_DIR}/ush:$ENV{PYTHONPATH}")
+
+
+###########################################################################
+# bufr to ioda tests:
+###########################################################################
+
+# prepare a test.yaml file from test.yaml.in by replacing
+# placeholder patterns __BUFRINPUTDIR__ and __IODAOUTPUTDIR__
+# with actual directory paths
+function(CREATE_CONFIG_FILE 
+	test_config_in 
+	test_config_out 
+	bufr_input_dir 
+	ioda_output_dir
+	ocean_basin_file
+)
+	file(READ "${test_config_in}" file_content)
+	string(REPLACE "__BUFRINPUTDIR__" "\"${bufr_input_dir}\"" temp_content "${file_content}")
+	string(REPLACE "__IODAOUTPUTDIR__" "\"${ioda_output_dir}\"" temp_content2 "${temp_content}")
+	string(REPLACE "__OCEANBASIN__" "\"${ocean_basin_file}\"" temp_content3 "${temp_content2}")
+	file(WRITE "${test_config_out}" "${temp_content3}")
+endfunction()
+
+
+set(TEST_WORKING_DIR ${PROJECT_BINARY_DIR}/test/soca)
+# message("bufr to ioda: TEST_WORKING_DIR = ${TEST_WORKING_DIR}")
```

Comment:
remove this line

Reply:
I pushed the changes. It tells me that parm/jcb-gdas is modified, but not committed (it's not my file), but it also tells me that there are no conflicts.